src/go/rpk/pkg/cli/cmd/topic/produce.go

```diff
@@ -203,7 +203,7 @@ Percent encoding reads into specific values of a record:
     %T    topic length
     %k    key
     %K    key length
-    %v    topic
+    %v    value
```

Comment:
NIT: In the commit message you could point to the formatter reference like
https://github.com/twmb/franz-go/blob/01d0c45d69d274d4fab20ab16f469d380f929462/pkg/kgo/record_formatter.go#L71

Reply:
(fwiw this was my mistake in the first place when introducing this typo)